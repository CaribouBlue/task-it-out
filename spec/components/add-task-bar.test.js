import React from 'react';
import renderer from 'react-test-renderer';
import AddTaskBar from 'src/components/add-task-bar';
import MockTasks from 'mock-data/tasks';

let consoleErrorSpy = null
beforeEach(() => {
  consoleErrorSpy = jest.spyOn(global.console, 'error')
})

afterEach(() => {
  expect(consoleErrorSpy).not.toHaveBeenCalled();
})

test('AddTaskBar component matches snapshot', () => {
  const component = renderer.create(
    <AddTaskBar></AddTaskBar>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
