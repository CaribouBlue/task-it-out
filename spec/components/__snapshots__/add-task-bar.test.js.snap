// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddTaskBar component matches snapshot 1`] = `
<form>
  <input
    aria-describedby="addTask"
    onChange={[Function]}
    onKeyDown={[Function]}
    placeholder="I need to task it out ..."
    type="text"
    value=""
  />
</form>
`;
